mozilla-central/setup

```diff
@@ -105,12 +105,12 @@ if [ -n "$INDEXED_GIT_REV" ]; then
 
     date
 
-    # And now process the downloads
-    for PLATFORM in linux64 macosx64 win64 android-armv7; do
-        date
-        echo "Processing platform $PLATFORM"
-        $CONFIG_REPO/shared/process-tc-artifacts.sh $PLATFORM
-    done
+    # And now process the downloads for each platform in parallel
+    (
+        for PLATFORM in linux64 macosx64 win64 android-armv7; do
```

Comment:
Just `echo "linux64\nmacosx64\nwin64\nandroid-armv7"`?

Or `echo "linux64 macosx64 win64 android-armv7" | tr " " "\n"`?

Reply:
Yeah that latter one seems cleaner, will update